### PR TITLE
Connect react redux

### PR DESCRIPTION
**Pull request by dvcorreia1984**
In this pull request the following was done: 

1. InitialState was added to booksSlice.js file
2. Content of Books state were made visible
3. Individual book component was moved to a seperate module
4. Functions to add and remove was added

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -1,38 +1,34 @@
...
-  const [author, setAuthor] = useState('');
+const Form = () => {
+  const dispatch = useDispatch(); // Initialize useDispatch
 
   const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const title = e.target.title.value;
+    const author = e.target.author.value;
+
```
Comment (julie-ify) [inline src/components/Form.js]: - [x] Kindly use the local state for your submissions. The values that should be sent to the store are the local states, not the direct HTML input values.

File: src/components/Form.js [hunk 2]
```diff
@@ -1,38 +1,34 @@
...
-          onChange={(e) => setTitle(e.target.value)}
-        />
-        <input
-          type="text"
-          placeholder="author"
-          value={author}
-          onChange={(e) => setAuthor(e.target.value)}
-        />
+        <input type="text" name="title" placeholder="title" />
+        <input type="text" name="author" placeholder="author" />
```
Comment (julie-ify) [inline src/components/Form.js]: - [x] The values of these inputs should be states, not the direct user inputs. The user inputs should be handled by an `onChange` method that sets the states relative to the inputs. Kindly add this.

File: src/components/BookList.js [hunk 3]
```diff
@@ -1,46 +1,13 @@
...
-    </div>
-  );
-};
+const BookList = () => (
+  <div>
+    <h2>Book List</h2>
+    <Book />
+    <Form />
+  </div>
+);
```
Comment (julie-ify) [inline src/components/BookList.js]: - [x] This is the component that should handle fetching the books from the store, not the other Book component you have. This component should use `useSelector` to fetch the books and then using map method and the props pass the data to the Book component. Kindly refactor your code.

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @dvcorreia1984,

Good job so far!

## Highlights!
- Initial books ✔️
- Descriptive README ✔️ 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [x] **[OPTIONAL]** This is an issue that should have been resolved in the first project. This is not a single-page application as we should expect. You nav links use the `<a>` whereas they should use `<NavLink>` or `<Link>`. The browser should not refresh when the user navigates through the pages.

  image

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dvcorreia1984** [comment]
Thank you @julie-ify . I applied all your comments. Very helpfull and decriptive comments.

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @dvcorreia1984 👋, 

Great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ books are accessed from inside the `<BookList />` with `useSelector` 📌 
✔️ No linter errors

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**